package.json

```diff
@@ -8,10 +8,8 @@
     "dev": "export DEBUG=dev && nodemon --exec babel-node server/index.js",
     "clean": "rm -rf build && mkdir build",
     "build": "npm run clean && babel -d ./build ./server",
-    "migrate": "node_modules/.bin/sequelize db:migrate",
-    "migrate:undo": "node_modules/.bin/sequelize db:migrate:undo",
-    "generate:model": "node_modules/.bin/sequelize model:generate",
-    "test": "NODE_ENV=test mocha --require @babel/register server/tests --exit"
+    "test": "NODE_ENV=test nyc mocha --require @babel/register server/tests --exit",
+    "coveralls": "nyc report --reporter=text-lcov | coveralls"
```

Comment:
Hey @chiboycalix Nice work with the coveralls setup. However, I think you overwrote the migration scripts. I'd suggest you do a rebase off the develop branch before pushing.

Reply:
Thanks for pointing this out @chuxmykel , It has been fixed